CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.4.0] - 2022-10-25
+### Added
+- Custom HTTP client support to Client configuration
+### Fixed
+- Additional support for complex domain parsing
```

Comment:
I think it would be helpful to add "for on-device-decisioning" to the end of this line item